examples/OktoResources/ResourceRelations/Connection/04.external-target-prometheus-insecure.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: blocks/v1beta1
+kind: Connection
+metadata:
+  name: connection-with-external-target
+  namespace: default
+  labels:
+    site.nbycomp.com/{{ .Values.placement.site.label }}: "true"
+    kind: K8sCluster
+spec:
+  externalTarget:
+    prometheus:
+      query: |-
+        {{ .Values.prometheus.query }}
```

Comment:
Let's write an actual query here, it can be short and silly. (and the same as in the other file)

Reply:
I would suggest to use one from the weighted queries examples.